### DISCUSSION TITLE
SVG asset file isn't getting displayed in readme

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [lowlighter/metrics#1548] [Q&A]
This might be a basic issue, but even with the full URL, it's not linking.

Let me explain:

I have these assets from various metrics workflows here: https://github.com/Anmol-Baranwal/Anmol-Baranwal/tree/main/assets.

There seems to be no issue with the actions and workflows.

![image](https://github.com/lowlighter/metrics/assets/74038190/c68c2415-d38b-4a04-96b7-5d2ef2ea50b0)

I even downloaded the SVG files, and they are displaying correctly, so the files are good:

![image](https://github.com/lowlighter/metrics/assets/74038190/6f2b760c-1df4-470d-bda2-035795e77ea2)

Now, I am trying to link the SVG assets in the readme:

![image](https://github.com/lowlighter/metrics/assets/74038190/75c0acfb-667b-4e51-b081-ade9a4de84f0)

I even tried this:

```
<img src="https://github.com/Anmol-Baranwal/Anmol-Baranwal/blob/main/assets/metrics.plugin.code.lines.svg" alt="hey" />
```

You can verify the URL, and it works, but the image is still not showing in the readme:

![image](https://github.com/lowlighter/metrics/assets/74038190/7879ac03-da9e-49df-aa15-69dcbc5b6148)

Now, we can insert the SVG files in the readme, but I want to do it this way. I'm not sure if the docs have been updated. I would really appreciate some help.

**Comment by Anmol-Baranwal** [comment]
@lowlighter 
I know you're busy, so please let me know whenever you have time what am I doing wrong.

**Comment by Vault108** [comment]
I would try  [reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links)

Example Code:

```
Other Content
[![achievements][achievements-image]][achievements-url]
[![followup][followup-image]][followup-url]

<!-- MARKDOWN LINKS & IMAGES -->
<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
[achievements-image]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.achievements.svg
[achievements-url]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.achievements.svg

[followup-image]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.followup.svg
[followup-url]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.followup.svg
More Content
```

Example Output: 

Other content
[![achievements][achievements-image]][achievements-url]
[![followup][followup-image]][followup-url]



[achievements-image]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.achievements.svg
[achievements-url]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.achievements.svg

[followup-image]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.followup.svg
[followup-url]: https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.followup.svg

More Content

**Reply by Anmol-Baranwal** [reply]
I will let you know if this works. Thanks for the information. I will mark it as answer as well if this resolves the issue.

**Reply by Anmol-Baranwal** [reply]
@Vault108 I tried, and it doesn't works. Even opening the URL in a new window doesn't works.

Still, thanks for the comment.

**Reply by Vault108** [reply]
Strange, the urls i provided where directly to your images. Now, I am willing to bet it does not like that there are several periods in the image URL.
I cloned your repo, and opend it in VSCode and previewed the readme, they did not display. I had to remove the periods and change them to underscores, but kept it a .svg Take a look at it https://github.com/Vault108/Anmol-Baranwal/tree/main/assets
Also upon looking at your repo, they display. So i am unable to see were you are seeing the issue? Is it locally?

**Reply by lowlighter** [reply]
I don't remember if it was patched by GitHub but I remember vaguely that if you used relative URLs (like you've used in the first time), it'll display correctly on the repository itself, but they may not appear on your profile since the urls won't be valid anymore.

I feel like the solution provided above should have worked indeed so it's rather strange.

Could you try again with the `<img>` syntax but using raw.githubusercontent instead ? (from your first post it seems you've put the regular github link)

Like:
```html
<img src="https://raw.githubusercontent.com/Anmol-Baranwal/Anmol-Baranwal/main/assets/metrics.plugin.achievements.svg">
```

**Reply by Anmol-Baranwal** [reply]
It's not even visible in the forked repo. I tried making the changes in a different branch.

![Image](https://github.com/lowlighter/metrics/assets/74038190/4c43721c-7b23-4a8f-91e3-e6bccb400251)

I attempted this:

![Image](https://github.com/lowlighter/metrics/assets/74038190/7a836984-3b1a-49c8-bd99-f73090e6c219)

![Image](https://github.com/lowlighter/metrics/assets/74038190/b9025b76-70b5-44f6-8ec3-eebc80e992c0)

I have, in fact, checked several repositories, and the metrics are not visible anywhere unless you use GitHub-deployed metrics.

So, please share the profile if it is visible anywhere.

**Reply by Anmol-Baranwal** [reply]
Also, I checked my profile from different account. It isn't visible :|
